examples/rbfe_single.py

```diff
@@ -0,0 +1,160 @@
+# This script computes the relative binding free energy of a single edge.
+
+import os
+import argparse
+import numpy as np
+
+from rdkit import Chem
+from rdkit.Chem import rdFMCS
+from rdkit.Chem import AllChem
+
+from fe import topology
+from md import builders
+from md import minimizer
+
+import functools
+
+from ff import Forcefield
+from ff.handlers.deserialize import deserialize_handlers
+
+import multiprocessing
+
+from fe import free_energy
+
+
+def wrap_method(args, fn):
+    gpu_idx = args[0]
+    os.environ['CUDA_VISIBLE_DEVICES'] = str(gpu_idx)
+    return fn(*args[1:])
+
+if __name__ == "__main__":
+
+    parser = argparse.ArgumentParser(
+        description="Relative Hydration Free Energy Consistency Testing",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+
+    parser.add_argument(
+        "--num_gpus",
+        type=int,
+        help="number of gpus",
+        required=True
+    )
+
+    parser.add_argument(
+        "--num_complex_windows",
+        type=int,
+        help="number of vacuum lambda windows",
+        required=True
+    )
+
+    parser.add_argument(
+        "--num_solvent_windows",
+        type=int,
+        help="number of solvent lambda windows",
+        required=True
+    )
+
+    parser.add_argument(
+        "--num_equil_steps",
+        type=int,
+        help="number of equilibration lambda windows",
```

Comment:
Should "lambda windows" in the next couple arguments be "MD steps per lambda window" or similar?

Reply:
Will fix!